apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/edge-connection-context.presenter.tsx

```diff
@@ -0,0 +1,51 @@
+import { IEdgeConnectionContext } from '@coscrad/api-interfaces';
+import { asFormattedMediaTimecodeString } from '@coscrad/media-player';
+import { Box, Typography } from '@mui/material';
+import { TimeRange } from '../../../../../components/resources/audio-item/audio-item-detail.full-view.presenter';
+import { convertMillisecondsToSeconds } from '../../../../../components/resources/utils/math';
+import { EdgeConnectionContextType } from './edge-connection-context-type.enum';
+
+const upperCaseFirstLetter = (str: string) => {
+    return str
+        .split(' ')
+        .map((word) => [word.charAt(0).toUpperCase(), word.slice(1)].join(''))
+        .join(' ');
+};
+
+type TimeRangeContext = {
+    type: EdgeConnectionContextType.timeRange;
+    timeRange: TimeRange;
+};
+
+interface EdgeConnectionContextPresenterProps {
+    context: IEdgeConnectionContext;
+}
+
+export const EdgeConnectionContextPresenter = ({
```

Comment:
I should have added the link. [Here](https://github.com/COSCRAD/coscrad/pull/430) is that PR. Look at the snippets in the conversation, as the diff is lost to history at this point (we didn't merge this).

Yes. Add a generic, optional `noteContext` property on the base `ResourceDetailFullViewPresenterProps` (note this may be linked to other interfaces \ generic constraints). If `noteContext` is specified in the detail presenter, you can respond by scrubbing to that point or highlighting that point on a timeline.

Reply:
I'm ok with this being future scoped for now in the sake of getting this work merged down sooner.

Reply:
Note that we have added a TODO for this now.